modules/home-manager/gtk.nix

```diff
@@ -105,7 +105,14 @@ in
         "gtk-4.0/gtk-dark.css".source = "${gtk4Dir}/gtk-dark.css";
       };
 
-    home.packages = lib.mkIf cfg.gnomeShellTheme [ pkgs.gnomeExtensions.user-themes ];
+    home = {
+      packages = lib.mkIf cfg.gnomeShellTheme [ pkgs.gnomeExtensions.user-themes ];
+
+      sessionVariables = lib.mkIf cfg.cursor.enable {
+        HYPRCURSOR_SIZE = "${toString cfg.size}";
+        HYPRCURSOR_THEME = "catppuccin-${cfg.cursor.flavor}-${cfg.cursor.accent}-cursors";
+      };
```

Comment:
i would prefer to keep this in the hyprland module, as there's not much reason for these to be set by default when only the gtk cursor is enabled